### DISCUSSION TITLE
Portable binaries for Notesnook

### DISCUSSION THREAD

**Discussion by ab1kenobee** [streetwriters/notesnook#993] [General]
Hello Members:
I am an EN long-term user.

Is Notesnook available as a PORTABLE?

THX.

**Comment by thecodrr** [comment]
Hey, not yet. Adding support for it is quite trivial though.

**Reply by ab1kenobee** [reply]
Thank you for a quick, firm answer.
Would please create a PORTABLE version yesterday?

I have been using EN since 2012... however due to the app's stagnation... I would like to test Notesnook.

I have installed Win 11 Lite on a new HP Omen 16 AMD R9 ... and would like to install as few apps to registry as possible.

Thank you much in advance.  ;)
Alan

**Comment by ab1kenobee** [comment]
Hello.
Has a PORTABLE version become available since my inquiry: SEPT 17 ?

**Reply by thecodrr** [reply]
Yes. The website hasn't been updated but you can download it from [the latest GitHub releases](https://github.com/streetwriters/notesnook/releases/tag/v2.2.1).
